pkg/imageengine/buildah/mount.go

```diff
@@ -66,23 +66,6 @@ func (engine *Engine) Mount(opts *options.MountOptions) ([]options.JSONMount, er
 			continue
 		}
 	}
-	//else {
-	//	builders, err := openBuilders(store)
```

Comment:
If these codes are not needed, could we delete it directly?

Reply:
Yes, it has been deleted in this pr. Check the modification type.